src/lib/utils.ts

```diff
@@ -94,19 +100,34 @@ const getConsoleTotalTable = ({
     targetBranchReport: squashReportByFileExtension(report.targetBranchReport)
   });
   return table.map(([ext, currentSize, targetSize]) => ({
-    Ext: `(${ext})`,
+    Ext: ext,
     [currentBranchName]: currentSize,
     [targetBranchName]: targetSize
   }));
 };
 
 const getFilesBreakDownTable = ({
   currentBranchName,
-  targetBranchName
-}: IPrintStdout): IConsoleTable => [
-  {
-    File: 'TODO:',
-    [currentBranchName]: 'currentBranchName',
-    [targetBranchName]: 'currentBranchName'
-  }
-];
+  targetBranchName,
+  report
+}: IPrintStdout): any =>
+  getFormattedRows(report).map(([fileName, currentSize, targetSize]) => ({
+    File: fileName,
+    [currentBranchName]: currentSize,
+    [targetBranchName]: targetSize
+  }));
+
+// TODO: fix types, and make them more robust
+const sortByDelta = (a: Array<string | number>, b: Array<string | number>) => {
```

Comment:
can the type of `a` and `b` be updated to `ITableRow` ?

Reply:
Unfortunately thet are not ITableRow, ITableRow is: 
```
ITableRow = [string, string, string]
```
while here we are dealing with:
```
 [string, number, number]
```
to allow delta calculation
this is why i left a TODO to eventually define those type in between the pipe trasformation